src/simsopt/configs/zoo.py

```diff
@@ -187,3 +195,44 @@ def get_w7x_data(Nt_coils=48, Nt_ma=10, ppp=2):
     ma.zs[:] = sZ[0:Nt_ma]
     ma.x = ma.get_dofs()
     return (curves, currents, ma)
+
+
+@SimsoptRequires(requests is not None, "You need to install the requests library to use this function. Run 'pip install requests'")
+def get_QUASR_data(ID, return_style='default'): 
+    """
+    Download a configuration from the QUASAR database.
+
+    Args:
+        ID: the ID of the configuration to download.  The database is navigatable at https://quasr.flatironinstitute.org/
+            Alternatively, you can download the latest full set of devices from https://zenodo.org/doi/10.5281/zenodo.10050655
+        
+        return_style: 'default' or 'json'. If 'default', the function will return the curves, currents and magnetic axis
+                      like the other configurations in the zoo. 
+                      If 'json' the function will return the full set of coils, magnetic axis and a number of surfaces
+                      parametrized in Boozer coordinates.
+    """
+    
+    assert return_style in ['default', 'json']
```

Comment:
Assertions can be disabled by caller/system config [e.g. if `PYTHONOPTIMIZE` is set], so it might be better not to rely on `assert` for a true runtime (not-debugging) check.